### ISSUE TITLE
Change button styling on data service

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1196]
**Please complete the following information**
Data repository URL: All
Data or metadata issue: Data service

Link: https://sdgdata.gov.uk/sdg-data/disaggregations.html

Reuse open-sdg green button for styling of buttons on data service

**Additional context**
Current colours don't meet constrast guidelines

**Comment by phillipgardiner** [comment]
@brockfanning Just incase its required: https://design-system.service.gov.uk/components/button/. The button colours and text colour should be the same as that used for the downloads on the indicator page.

**Comment by brockfanning** [comment]
@phillipgardiner I believe this is also done as part of https://github.com/open-sdg/sdg-build/pull/262.
